portal-web/test/functional/com/liferay/portalweb/paths/portlet/frontenddataset/DataSet.path

```diff
@@ -484,7 +494,11 @@
 	<td>xpath=(//a[contains(@class,'btn')][contains(@href,'new') or contains(@href,'redirect') or contains(@href,'mvc') or contains(@href,'javascript')][.//*[name()='svg'][contains(@class,'icon-plus')]][not(contains(@class,'borderless'))] | //button[not(contains(@class,'borderless'))][not(contains(@class,'btn-unstyled'))][*[contains(@class,'icon-plus')]])[2]</td>
 	<td></td>
 </tr>
-
+<tr>
+	<td>RADIO_BUTTON_OPTION</td>
+	<td>//span[contains(., '${key_optionName}')]/preceding-sibling::input</td>
+	<td></td>
+</tr>
```

Comment:
Can we use a generic radio button path instead of creating a new one? If not, I'm ok with creating it.